runtime/lua/vim/lsp/buf.lua

```diff
@@ -153,7 +202,7 @@ end
 --- vim.api.nvim_command [[autocmd CursorMoved <buffer> lua vim.lsp.buf.clear_references()]]
 --- </pre>
 function M.document_highlight()
-  local params = util.make_position_params()
+  local params = util.jkmake_position_params()
```

Comment:
Is that intended?

Reply:
Sorry, pushed a wrong commit. It's fixed now